tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -146,10 +146,10 @@
     <!-- Latest package https://www.nuget.org/packages/NUnit/3.13.3 -->
     <PackageReference Include="NUnit" Version="3.13.3" />
 
-    <!-- Integration: NUnit3.TestAdapter -->
-    <!-- Assembly: NUnit3.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.3.1 -->
-    <PackageReference Include="NUnit3TestAdapter" Version="4.3.1" />
```

Comment:
I don't know why this one ups and leaves? 😬

Reply:
fixed by the merge commit.